src/data/live/data-source.js

```diff
@@ -0,0 +1,56 @@
+// import { dataSource as baseLive } from '@apollosproject/data-connector-church-online';
+import RockApolloDataSource from '@apollosproject/rock-apollo-data-source';
+import ApollosConfig from '@apollosproject/config';
+
+const { CHURCH_ONLINE, ROCK_MAPPINGS } = ApollosConfig;
+
+export default class LiveStream extends RockApolloDataSource {
+  resource = 'LiveStream';
+
+  mediaUrls = () => {
+    return CHURCH_ONLINE.MEDIA_URLS;
```

Comment:
we shouldn't make any references to CHURCH_ONLINE right?

Reply:
I kept those because I don't think we're ready for totally moving away from Church Online quite yet. I think this is just step 1 in getting there. We still have to change over to use the native player in the app, right?

Reply:
I mean, if that capability is there then yeah, we can get rid of the Church Online references.

Reply:
ok that makes sense, I believe we only use `webviewurls` not media, so we shouldn't set both if we don't use both.

Reply:
Yep, I can dump `mediaUrls` from here and from the `config.yml` technically.

Reply:
I can just return `null` for `mediaUrls` too.